src/Examples/UnitModels/Reactors/equilibrium_reactor_testing.ipynb

```diff
@@ -0,0 +1,1135 @@
+   "source": [
+    "m.fs.s03 = Arc(source=m.fs.M101.outlet, destination=m.fs.C101.inlet)\n",
+    "m.fs.s04 = Arc(source=m.fs.C101.outlet, destination=m.fs.H101.inlet)\n",
+    "m.fs.s05 = Arc(source=m.fs.H101.outlet, destination=m.fs.R101.inlet)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We have now connected the unit model block using the arcs. However, each of these arcs link to ports on the two unit models that are connected. In this case, the ports consist of the state variables that need to be linked between the unit models. Pyomo provides a convenient method to write these equality constraints for us between two ports and this is done as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "TransformationFactory(\"network.expand_arcs\").apply_to(m)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Adding expressions to compute operating costs\n",
+    "\n",
+    "In this section, we will add a few Expressions that allows us to evaluate the performance. Expressions provide a convenient way of calculating certain values that are a function of the variables defined in the model. For more details on Expressions, please refer to: https://pyomo.readthedocs.io/en/latest/pyomo_modeling_components/Expressions.html\n",
+    "\n",
+    "For this flowsheet, we are interested in computing hydrogen production in millions of pounds per year, as well as the total costs due to pressurizing, cooling and heating utilities:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let us first add an Expression to convert the product flow from mol/s to MM lb/year of hydrogen. We see that our molecular weight exists in the thermophysical property package, so we may use that value for our calculations."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "m.fs.hyd_prod = Expression(expr=pyunits.convert(m.fs.R101.outlet.flow_mol[0]*m.fs.R101.outlet.mole_frac_comp[0, \"H2\"]\n",
+    "                                               *m.fs.thermo_params.H2.mw, # MW defined in properties as kg/mol\n",
+    "                                               to_units=pyunits.Mlb/pyunits.yr)) # converting kg/s to MM lb/year"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, let us add expressions to compute the reactor cooling cost (\\\\$/s) assuming a cost of 0.212E-4 \\\\$/kW, the compression cost (\\\\$/s) assuming 0.12E-2 \\\\$/kW, and the heating utility cost (\\\\$/s) assuming 2.2E-4 \\\\$/kW. Note that the heat duty is in units of watt (J/s). The total operating cost will be the sum of the costs, expressed in \\\\$/year assuming 8000 operating hours per year (~10\\% downtime, which is fairly common for small scale chemical plants):"
```

Comment:
Expressing scientific notation like "0.212E-4" in text doesn't look good. (Also, why not 2.12E-5?)